File: python/10101번.py
a=[]
for i in range(3):
    a.append(int(input()))
if sum(a)!=180:
    print("Error")
else:
    if a[0]==a[1]==a[2]:
        print("Equilateral")
    elif a[0]==a[1] or a[1]==a[2] or a[0]==a[2]:
        print("Isosceles")
    else:
        print("Scalene")

File: java/10101번.java
import java.io.IOException;
import java.util.*;
import java.math.BigInteger;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int A=sc.nextInt();
        int B=sc.nextInt();
        int C=sc.nextInt();
        String res = "";
        if(A==60&&B==60&&C==60){
            res="Equilateral";
        }else if(A+B+C==180){
            if(A==B||B==C||A==C){
                res="Isosceles";
            }else{
                res="Scalene";
            }
        }else{
            res="Error";
        }
        System.out.println(res);
    }
}